x-pack/plugins/siem/server/lib/overview/query.dsl.ts

```diff
@@ -30,7 +31,7 @@ export const buildOverviewNetworkQuery = ({
 
   const dslQuery = {
     allowNoIndices: true,
-    index: [auditbeatAlias, logAlias, packetbeatAlias],
+    index: [auditbeatAlias, logAlias, packetbeatAlias, winbeatAlias],
```

Comment:
Did you want this added to the `overview` query as well? It wasn't mentioned in the description, so just checking.

Reply:
The overview has very specific beats its looking at, so not at this time I would say.